app/script/view_model/panel/ConversationDetailsViewModel.js

```diff
@@ -132,32 +132,32 @@ z.viewModel.panel.ConversationDetailsViewModel = class ConversationDetailsViewMo
 
     this.showActionClear = ko.pureComputed(() => this.activeConversation() && this.activeConversation().isClearable());
 
-    this.showActionGuestOptions = ko.pureComputed(() => {
-      return this.isActiveGroupParticipant() && this.activeConversation() && this.activeConversation().inTeam();
-    });
-
     this.showActionLeave = ko.pureComputed(() => this.activeConversation() && this.activeConversation().isLeavable());
 
     this.showActionMute = ko.pureComputed(() => {
       return this.activeConversation() && this.activeConversation().isMutable() && !this.isTeam();
     });
 
-    this.showActionNotificationSettings = ko.pureComputed(() => {
+    this.showOptionGuests = ko.pureComputed(() => {
+      return this.isActiveGroupParticipant() && this.activeConversation().inTeam();
```

Comment:
Is it safe to remove the `&& this.activeConversation()`?

Reply:
I believe so. The active group participants checks for it as well and should thus never return true if active conversation is not set.

Reply:
Ah ok, it's included in `isActiveGroupParticipant`